.github/workflows/dev.yml

```diff
@@ -0,0 +1,13 @@
+name: Dev-Build
+
+on:
+  push:
+  schedule:
+    - cron:  '21 21 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: axonivy-market/github-workflows/.github/workflows/ci.yml@v2
+    with:
+      mvnArgs: '"-Divy.engine.download.url=https://dev.axonivy.com/permalink/dev/axonivy-engine.zip" "-Divy.engine.version=(10.0.0,]"'
```

Comment:
runs the tests with the latest engine ... 
@alexsuter  @ivy-cst 
I think I'll add this as pipeline to:
- axonivy-market/github-workflows
- add a reference to it into your product-template
- add a reference to all existing products.